packages/desktop/src/components/channel-message.js

```diff
@@ -1371,7 +1393,7 @@ const RepliedMessage = ({ message, getMember }) => {
           />
         ) : (
           <Avatar
-            url={authorMember?.profilePicture.small}
+            url={authorMember?.profilePicture?.small}
```

Comment:
What will the app `authorMember` data look like?

Reply:
for now I went with an empty author to avoid adding more complexity fetching apps' avatars and whatnot. this is what a webhook message and reply look like:

<img width="604" alt="Screenshot 2022-06-03 at 13 39 10" src="https://user-images.githubusercontent.com/314617/171846691-4eac0ec6-b0e3-4aaf-b15b-11505838ea72.png">